### ISSUE TITLE
Front and center and impossible to miss - Parameter Group vs Options Group doc page

### ISSUE THREAD

**Issue by matthewcummings** [awsdocs/amazon-rds-user-guide#126]
I work with RDS a good amount, but I work with lots of other AWS and non-AWS services and tools. Every time I come back to Option groups and/or Parameter groups, I always forget what the distinction is between the two and I end up here https://www.reddit.com/r/aws/comments/5r0l8e/what_is_the_difference_between_rds_option_groups/.

I would be helpful if you had another page here https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_RDS_Configuring.html, something short, literally titled Option groups vs Parameter groups. I suspect that I'm not alone. I am certain that users who are new to RDS will benefit from this.

**Comment by matthewcummings** [comment]
BTW, I really appreciate the attention my various feature and doc request tickets have been receiving from AWS. I live and breathe AWS and while I am a huge fan of the platform overall, there are lots of gotchas and inconsistencies with the platform.

tl;dr great to see that you (collectively speaking) are willing to keep updating the doc and improve service offerings.

**Comment by randyurbano** [comment]
We appreciate your feedback. I'll look into this issue as soon as I can.

**Comment by randyurbano** [comment]
I added information about option groups and DB parameter groups to the topic:

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_RDS_Configuring.html

**Comment by randyurbano** [comment]
Thanks again for your feedback.

**Comment by matthewcummings** [comment]
You're welcome @randyurbano 
Question - is there a GitHub board for RDS feature requests?

**Comment by rleibbrandt-aws** [comment]
Hey Matthew - Randy pinged me on this.  We don't have a board for RDS feature requests at this point.  It is something we're looking into.  Our support teams, technical account managers, solution architects and account managers have access to a system where they can log customer feature requests.   Do you have an account team that you work with?

**Comment by matthewcummings** [comment]
@rleibbrandt-aws I do not have an account team currently, unfortunately. :(

Just a heads up though, what I wanted to post/suggest is a bit high level and probably something already on your radar. I have been working with AWS full time for about 6-7 years at this point and what I'd love to see is full fledged IAM support with Aurora. I mainly use Postgres, but it would be great to see with MySQL Aurora as well

I've had to write custom scripts and Terraform modules to work around the handoff between AWS APIs and Postgres' interface i.e. when I provision RDS instances, it's an extra pain to manage the (logical) databases running on the instances. Put another way, it would be wonderful if I could fully manage database resources via IAM the way you can with DynamoDB. I'd love to be able to map a role to a Lambda/task/pod/instance to a specific RDS database role. And be able to provision Postgres roles via the AWS SDK, etc etc.

RDS currently offers limited IAM support, which is great but it would be amazing if we could literally stop managing RDS credentials altogether and do things the "native AWS way".  Let me know if you need further details/clarification.
